lang/cs/Org.Apache.REEF.Wake.Tests/PrefixedWritableString.cs

```diff
@@ -0,0 +1,109 @@
+﻿/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Org.Apache.REEF.Tang.Annotations;
+using Org.Apache.REEF.Wake.Remote;
+
+namespace Org.Apache.REEF.Wake.Tests
+{
+    [NamedParameter("identifier in PrefixedWritable")]
+    public class StringId : Name<int>
+    {
+    }
+    
+    
+    /// <summary>
+    /// Writable wrapper around the string class which takes integer prefix
```

Comment:
Yes, it is not used.

On Wed, May 13, 2015 at 9:14 AM, Markus Weimer <notifications@github.com>
wrote:

> In lang/cs/Org.Apache.REEF.Wake.Tests/PrefixedWritableString.cs
> <https://github.com/apache/incubator-reef/pull/181#discussion_r30247731>:
>
> > +using System;
> > +using System.Threading;
> > +using System.Threading.Tasks;
> > +using Org.Apache.REEF.Tang.Annotations;
> > +using Org.Apache.REEF.Wake.Remote;
> > +
> > +namespace Org.Apache.REEF.Wake.Tests
> > +{
> > +    [NamedParameter("identifier in PrefixedWritable")]
> > +    public class StringId : Name<int>
> > +    {
> > +    }
> > +
> > +
> > +    /// <summary>
> > +    /// Writable wrapper around the string class which takes integer prefix
>
> The int isn't used in serializing, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/181/files#r30247731>.
>